pages/jdl.md

```diff
@@ -891,8 +891,8 @@ Here are the application options supported in the JDL:
   <tr>
     <td>cacheProvider</td>
     <td>ehcache or hazelcast</td>
-    <td>ehcache, hazelcast, infinispan, no</td>
-    <td>ehcache for monoliths and gateways, hazelcast otherwise</td>
+    <td>ehcache, caffeine, hazelcast, infinispan, no</td>
+    <td>ehcache or caffeine for monoliths and gateways, hazelcast otherwise</td>
```

Comment:
I think this comment is an explanation for the default value, so caffeine should not be added